google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -515,7 +527,7 @@ def get_table_names(self, connection, schema=None, **kw):
 FROM information_schema.tables
 WHERE table_schema = '{}'
 """.format(
-            schema
+            schema or ""
```

Comment:
By default `schema` arg is `None`, so it tried to search for tables with schema `'None'`. Not sure how I've written this, it's not smart at all. Fixing it.